### PR TITLE
fix update of TCPMetrics

### PR DESCRIPTION
**Pull request by andrekassis**
added fix for tcp metrics

### CODE REVIEW

File: flow/table.go [hunk 1]
```diff
@@ -420,7 +427,18 @@ func (ft *Table) processFlowOP(op *Operation) {
 		fl.Metric.BAPackets += op.Flow.Metric.BAPackets
 
 		fl.Last = op.Flow.Last
-
+		if fl.Transport != nil {
```
Comment (safchain) [inline flow/table.go]: `if fl.Transport != nil && fl.Transport.Protocol == FlowProtocol_TCP && fl.TCPMetric != nil` 

Can you remove my `NOTE` line 422

Comment (andrekassis) [reply flow/table.go]: done

**Comment by andrekassis** [comment]
run kydive-functional-tests-backend-orientdb

**Comment by andrekassis** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by andrekassis** [comment]
run skydive-functional-tests-backend-orientdb

**Pull request merged**